ai-worker/app/resumes/services/vector.py

```
@@ -0,0 +1,25 @@
+from langchain_openai import OpenAIEmbeddings
+from app.core.config import settings
+
+class VectorService:
+    def __init__(self):
+        try:
+            self.embeddings = OpenAIEmbeddings(
+                model=settings.OPENAI_EMBEDDING_MODEL,
+                openai_api_key=settings.OPENAI_API_KEY
+            )
+        except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)